modules/network/src/main/java/org/apache/ignite/network/DefaultMessagingService.java

```diff
@@ -82,12 +82,12 @@ public class DefaultMessagingService extends AbstractMessagingService {
     private final AtomicLong correlationIdGenerator = new AtomicLong();
 
     /** Executor for outbound messages. */
-    private final ExecutorService outboundExecutor = Executors.newSingleThreadExecutor(
+    private final ExecutorService outboundExecutor = Executors.newFixedThreadPool(2,
             new NamedThreadFactory("MessagingService-outbound-", LOG)
     );
 
     /** Executor for inbound messages. */
-    private final ExecutorService inboundExecutor = Executors.newSingleThreadExecutor(
+    private final ExecutorService inboundExecutor = Executors.newFixedThreadPool(2,
```

Comment:
Why specifically 2?

Reply:
Maybe, in the future, we should make this pool a little bit bigger